pkg/ccl/backupccl/manifest_handling.go

```diff
@@ -722,6 +722,7 @@ func resolveBackupManifests(
 	mkStore cloud.ExternalStorageFromURIFactory,
 	from [][]string,
 	incFrom []string,
+	defaultIncFrom []string,
```

Comment:
given that either len(incFrom) >0 or len(defaultIncFrom)>0, but never both, I wonder if there's a way to refactor this function so only incFrom gets passed in. It seems the logic that makes this hard is the following, and I don't quite understand it:
```
prev, err = FindPriorBackups(ctx, incStores[0], IncludeManifest)
			if len(prev) > 0 {
				incFrom = defaultIncFrom
			} else {
				incFrom = from[0]
				incStores = baseStores
				prev, err = FindPriorBackups(ctx, incStores[0], IncludeManifest)
			}
```
In words, is this saying: first check the default incremental location, and if there aren't any, check the full backup location?